gateway-clusters/ibm-external-compute-job.yaml

```diff
@@ -72,7 +72,7 @@ spec:
       - name: ibm-external-compute-image-pull
       containers:
       - name: provision
-        image: &image us.icr.io/armada-master/stranger:v1.0.9
+        image: &image us.icr.io/armada-master/stranger:v1.0.14
```

Comment:
Hello! I think it needs to be armada-stranger:v1.0.14. What do you think? Thank you!

Reply:
Right. Thank you :)